### DISCUSSION TITLE
Change filament menu not appearing

### DISCUSSION THREAD

**Discussion by hayhay7789** [Jyers/Marlin#730] [Q&A] [answered]
Had to switch from a precompiled file to a custom one to accommodate my Micro Swiss X and Y offsets. I'm now finding that the "change filament" menu option no longer appears on my E3V2. Is there an option to enable it? I've checked config and config_adv for M600, filament, load, and unload with Ctrl+F in Visual Studio Code but haven't been able to find anything which would seem to explain my issue.
[Configuration.zip](https://github.com/Jyers/Marlin/files/6438484/Configuration.zip)

**Comment by Cal-LF** [comment, marked as answer]
Are you using one of the E3V2 templates in the Configuration Files? They will define Nozzle Park and Advanced Pause which are needed for filament change.

**Reply by hayhay7789** [reply]
Yes I am, thanks for the tip.

Solved - mistakenly forgot to uncomment a title to enable the manual levelling
